packages/@manta-style/plugin-system/docs/MockPlugin.md

```diff
@@ -4,7 +4,115 @@
 - Package name for a mock plugin should be `@manta-style/plugin-mock-*`
 
 ## Node type supported
+
 - ArrayType
 - StringType
 - NumberType
-- BooleanType
\ No newline at end of file
+- BooleanType
+
+## Plugin Interface
+
+```typescript
+type SupportedMockNodeType =
+  | 'ArrayType'
+  | 'StringType'
+  | 'NumberType'
+  | 'BooleanType';
+
+type Plugin = {
+  name: string;
+  mock: {
+    [key in SupportedMockNodeType]?: (
+      type: Type,
+      annotations: Annotation[],
+    ) => any
+  };
+};
+```
```

Comment:
actually, for StringType the return should be `string | null`, NumberType return should be `number | null`, etc.